models/agent_helpers.go

```diff
@@ -342,6 +342,107 @@ func ChangeAgent(q *reform.Querier, agentID string, params *ChangeCommonAgentPar
 	return row, nil
 }
 
+// FindPMMAgentsForNode gets pmm-agents for node where it runs.
+func FindPMMAgentsForNode(q *reform.Querier, nodeID string) ([]*Agent, error) {
+	structs, err := q.SelectAllFrom(AgentTable, "WHERE runs_on_node_id = $1", nodeID)
+	if err != nil {
+		return nil, status.Errorf(codes.FailedPrecondition, err.Error())
+	}
+
+	var res []*Agent
+	for _, str := range structs {
+		row := str.(*Agent)
+		if row.AgentType == PMMAgentType {
```

Comment:
Let's add it to SQL query

Reply:
And here https://github.com/percona/pmm-managed/pull/199/files#diff-91021c9842093db76266b0e5212c3f5aR415